ldscript.txt

```diff
@@ -68,6 +68,9 @@ SECTIONS
         build/sa2/asm/game.o(.text);
         build/sa2/src/game/player_callbacks_1.o(.text);
         build/sa2/asm/game_2.o(.text);
+        build/sa2/asm/stage_ui.o(.text);
+        build/sa2/asm/stage_loading_screen.o(.text);
```

Comment:
stage_intro?

Not a fan of `screen` in module names

Reply:
Sure, I can switch it.

What's the reason for that?